onnxruntime/test/framework/session_state_test.cc

```diff
@@ -100,7 +100,7 @@ TEST_P(SessionStateAddGetKernelTest, AddGetKernelTest) {
 
 INSTANTIATE_TEST_SUITE_P(SessionStateTests, SessionStateAddGetKernelTest, testing::Values(0, 1));
 
-namespace {
+namespace anon {
```

Comment:
nit: if "anon" is short for anonymous, it's not that anymore.
can it just be put in the enclosing namespace with a comment indicating why it can't be in an unnamed namespace?